### PR TITLE
OpenApiService: export multi method on single path

### PR DESCRIPTION
**Pull request by trejjam**
This PR extend exist `pathItem` instead of replacing it.

### CODE REVIEW

File: src/OpenApiService.php [hunk 1]
```diff
@@ -44,7 +44,11 @@ public function createSchema()
 		foreach ($this->getEndpoints() as $endpoint) {
 			$endpointId++;
 
-			$pathItem = new PathItem();
+			$pathItem = $paths->getPath($endpoint->getMask());
+			if ($pathItem === NULL) {
```
Comment (f3l1x) [inline src/OpenApiService.php:48]: Is this path used anywhere?

Comment (trejjam) [reply src/OpenApiService.php:48]: What do you mean?

Path (from `paths`) is filled by `$endpoint->getMask()`. If you create a method for each `GET`|`POST`|... using the same mask, then only last endpoint is exported using `OpenApiService`.

Comment (f3l1x) [reply src/OpenApiService.php:48]: Sorry, I ment, is variable `$pathItem` used anywhere? I don't see it.

Comment (trejjam) [reply src/OpenApiService.php:48]: Yes, [here](https://github.com/Scalesoft/openapi/blob/1aac536a32fdf793ee46d9f48ce79b124ee736f8/src/OpenApiService.php#L129)

Comment (f3l1x) [reply src/OpenApiService.php:48]: Oh, thanks.

**Pull request merged**